chain/network/src/accounts_data/mod.rs

```diff
@@ -51,17 +51,17 @@ pub(crate) enum Error {
 
 /// Most up-to-date AccountData of this node and a signer
 /// to sign it with when there is a need to override some
-/// already signed data received from the network. See `Cache::set_local`
+/// already signed data received from the network. See `AccountDataCache::set_local`
 /// for more details.
```

Comment:
nit: The line lenth seems a bit wobbly here, in vs code I use the rewrap extension to make it neat automatically.